pkg/ccl/changefeedccl/cdcevent/settings.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package cdcevent
+
+import (
+	"time"
+
+	"github.com/cockroachdb/cockroach/pkg/settings"
+)
+
+// RowFetcherTraceKVs controls if we write the experimental new format BACKUP
+// metadata file.
+var RowFetcherTraceKVs = settings.RegisterBoolSetting(
```

Comment:
I'm actually wondering if having these settings is the right thing to do.
We are (ab)using these settings in great many places.
These are intended to debug low level failures.
Could the same functionality be accomplished via vlog?
For example, we can say "vmodule=rowfetcher_cache=3" ...
When we return fetcher, we can check `if log.V(3)` ... enable tracing.

Reply:
I don't feel too strongly about using settings vs `if log.V(3)`. I do think that having a setting for the frequency is a no brainer (it's certainly a problem if we start logging too many rows / sec right?). But TraceKV can be enabled with the log.V(3) bool.

Reply:
even frequency doesn't need to be enabled here -- as long as you can set frequence as you do with TraceKV fetcher option.  It's fine to keep it a constant (unless we find that we absolutely need to be able to control it dynamically)

Reply:
I feel like there's cases where you want to change the frequency. If a changefeed has 300M rows, then limiting to 1s is good. If it only has like 3 rows, then we should print all the KVs so we don't miss anything.